MetaMask/metamask-mobile
# Fix: update PPOM Validator to address blockaid performance issues

## **Description**
Update pom-validator.
Latest update of PPOM has performance improvements which are need for mobile app.

## **Related issues**

Fixes: MetaMask/MetaMask-planning#1555

## **Manual testing steps**

1. Open the app
2. Go to test DAPP
3. Send malicious transaction
4. It should show alert

## **Screenshots/Recordings**

NA

### **Before**

NA

### **After**

NA

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
